ducc/lib/conversion.go

```diff
@@ -518,6 +518,91 @@ func CreateThinImage(manifest da.Manifest, layerLocations map[string]string, inp
 	return nil
 }
 
+func PushImageToRegistry(outputImage Image) (err error) {
+	// is necessary this mechanism to pass the authentication to the
```

Comment:
I think I don't understand the comment.

Reply:
This and the above two comments refers to part of legacy code, I didn't modify them. They just needed restructuring to incorporate podman integration.

Reply:
Ok, let's still use the opportunity to improve the code (@siscia perhaps you can help)

Reply:
The docker documentation says that it is not necessary to pass the authentication credential. If we don't pass the credentials it does not work. It is actually necessary to pass the credentials.

Reply:
Understood, thanks!  Let's take this as a code comment :)